docs/ci/github.mdx

```diff
@@ -173,3 +173,184 @@ The `shorebird-patch` action also outputs the patch number:
 ```
 
 :::
+
+# 🫸 Push to Deploy
```

Comment:
```suggestion
## 🫸 Push to Deploy
```

I think we need to change all the headers to be nested beneath the top-level GitHub header